.github/workflows/integration-tests-pr.yml

```diff
@@ -4,7 +4,7 @@ on:
     inputs:
       tests:
         description: 'The tests to run.'
-        required: true
+        required: false
```

Comment:
It's not required, I think, right? When it's left empty, all tests will run

Reply:
idk why it is showing as true, it is true when you edit the file